core/lib/types/src/l2_to_l1_log.rs

```diff
@@ -63,6 +55,22 @@ impl L2ToL1Log {
     }
 }
 
+/// Returns the minimum number of items in the Merkle tree built from L2-to-L1 logs
+/// for a certain protocol version.
+pub fn min_l2_to_l1_logs_tree_size(protocol_version: ProtocolVersionId) -> usize {
```

Comment:
Maybe it makes sense to make it a static method on `L2ToL1Log`?
E.g. `L2ToL1Log::min_tree_size(protocol_version)`.

Reply:
@Deniallugo suggested vice versa https://github.com/matter-labs/zksync-era/pull/1234#discussion_r1500913056, i.e. to replace static method with a function. I don't have strong opinion, IMO both variants are ok

Reply:
lol, so be it